82-get-the-google-cache-age.md

```diff
@@ -0,0 +1,19 @@
+# How can you Get the Google cache age of any URL or web page?
+
+You only need to use the following URL format:
+
+```python
+http://webcache.googleusercontent.com/search?q=cache:URL
+```
+
+Be sure to replace `URL` with the proper web address of the page or site whose cache you want to retrieve and see the time for.
+
+`Example:`
+
+```python
+http://webcache.googleusercontent.com/search?q=cache:eliprogramming.com
```

Comment:
Does not work for me 
![image](https://user-images.githubusercontent.com/12773753/126868411-2043205c-ceb1-49ac-8fbc-657c09ea4782.png)